commit 3aaee86ee33af276d2e879f5a645cc6dc850de22
Author: Felipe Pena <felipensp@gmail.com>
Date:   Sat Oct 5 12:52:55 2013 -0300

    - Fixed bug #65818 (Segfault with built-in webserver and chunked transfer encoding)

--- php5.orig/sapi/cli/php_cli_server.c
+++ php5/sapi/cli/php_cli_server.c
@@ -1555,6 +1555,7 @@ static int php_cli_server_client_read_re
 		}
 		client->request.content_len = 0;
 	}
+	client->request.content = perealloc(client->request.content, client->request.content_len + length, 1);
 	memmove(client->request.content + client->request.content_len, at, length);
 	client->request.content_len += length;
 	return 0;
